src/index.js

```diff
@@ -172,7 +172,8 @@ export default (function create(defaults) {
 		return fetchFunc(url, {
 			method: options.method,
 			body: data,
-			headers: deepMerge(options.headers, customHeaders, true)
+			headers: deepMerge(options.headers, customHeaders, true),
+			credentials: options.withCredentials === true ? 'include' : 'same-origin'
```

Comment:
As every byte matters, I'm suggesting a shorter solution:
``` JavaScript
credentials: options.withCredentials && 'include'
```
If `options.withCredentials` is `true` it will assign 'include' otherwise, the default option will be chosen which is 'same-origin'.

Also if this suggestion is somehow incorrect, you can make it shorter by simply removing `=== true`.

Reply:
Good point, I will update the PR. Thanks!